lib/Shippo/ApiRequestor.php

```diff
@@ -155,7 +155,11 @@ private function _interpretResponse($rbody, $rcode)
         }
         return $resp;
     }
-    
+
+    private function _getAuthorizationType($apiKey = '')
+    {
+        return strpos($apiKey, 'oauth.') === 0 ? 'Bearer' : 'ShippoToken';
```

Comment:
Is this line tested somewhere?

Reply:
No

Reply:
added unit test in https://github.com/Weebly/shippo-php-client/pull/1/commits/ab2d5368f96c0734dde56b9ea5403fdea4692753